test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,9 +1,55 @@
+---
 # NXAPI is enabled differently on NX-OS platforms.
 # Try both commands to enable NXAPI and ignore any errors.
 - name: enable nxapi on remote device
   nxos_config:
     lines:
-      - feature cli
-      - feature cli sandbox
+      - feature nxapi
+      - feature nxapi sandbox
```

Comment:
Thinking more however.. I don't think that we need the sandbox for Ansible testing.  I believe just `feature nxapi` should be sufficient.  I will confirm and then remove `sandbox` if possible.